tests/helics/add_subdirectory_tests/main-apps.cpp

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Is there any way we can have these files be the same as in the other tests?  I was working on adding the package_config_tests as well which would use the same files again, just wondering if we could use the same files over again instead of adding more copies,  or are they a little different?

Reply:
The CMakeLists.txt files are different, but the source files are the same.

Reply:
Lets work on unifying them in the next version when we merge in the package_config tests which will also be similar.  For now this looks fine.  Though I am wondering if these tests belong in here or in a different repository.    Is there a way we can make sure they don't get run if someone does make test?

Reply:
The way I know of would be only enabling them with an option that defaults to off, maybe the cpack enable option. I need to see if I can find a list of ctest properties for one that disables a test by default.

I think the find_package tests should be here as a quick test to catch some possible issues with HELICSConfig and the export targets to reduce the chance of triggering builds in a lot of dependent repositories that will all fail.

Reply:
I set the first slower add_subdirectory test to disabled, and the find_package test to enabled since it only takes a few seconds to run.

Reply:
Is the add_subdirectory test still running on travis?

Reply:
Yes, it still runs on Travis

Reply:
But the reason it still runs is because the CMake version used for that Travis build doesn't support the DISABLED property. :neutral_face:

Reply:
that is unfortunate, sounds like that might be the best option to add a specific advanced flag to enable that test

Reply:
Yea. The DISABLED property doesn’t make much sense either — I haven’t found any arguments to ctest that will run a disabled test. The way I can see to use it is setting the disabled flag if eg the python interface is disabled, so that the tests not run still show up in the summary after running ctest.